### PR TITLE
Feature: Layers-using endpoint / function.

### PR DESCRIPTION
**Pull request by ryan-roemer**
- Adds layer serverless configuration and new endpoint to show how it works.
- Add versions to `terraform-aws-serverless` modules to pin
- Separate out the base stuff from all the optional stuff so folks starting from this repo can get rid of everything they don't need more easily...

/cc @tptee

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -12,7 +12,8 @@ terraform {
 # Base `serverless` IAM support.
 ###############################################################################
 module "serverless" {
-  source = "FormidableLabs/serverless/aws"
+  source  = "FormidableLabs/serverless/aws"
+  version = "0.5.3"
```
Comment (tptee) [inline terraform/main.tf]: Should we pin this to the latest `0.6.0`?

**Review by tptee — APPROVED** [review]
Looks great!!

File: terraform/main.tf (hunk 1 above)
Comment (ryan-roemer) [reply terraform/main.tf]: Yep. I'll update and merge.

Comment (ryan-roemer) [reply terraform/main.tf]: Done.

**Comment by ryan-roemer** [comment]
@tptee -- I apologize. I had unpushed commits. Mind a quick "one more" review? Thanks!!!

**Review by tptee — APPROVED** [review]
New commits look ready to go!

**Pull request merged**